File: 프로그래머스/0/120884. 치킨 쿠폰/치킨 쿠폰.py
def solution(chicken):
    answer = chicken // 10
    c = chicken // 10 + chicken % 10
    while c >= 10:                    # 참일 동안만 반복한다.
        a = c // 10
        b = c % 10
        answer += a
        c = a + b
    return answer

File: 프로그래머스/0/120884. 치킨 쿠폰/치킨 쿠폰.java
class Solution {
    public static int solution(int chicken) {
        int div = chicken;
        int answer = 0;

        while (div >= 10){
        int mod = div % 10;
        div = div/10;
        answer += div;
        div = div + mod;

        }
        return answer;
    }
}